src/components/Pages/ServicesPage/ServicesHeader.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import {Link} from "react-router-dom";
+
+import { OnMobileTablet, OnDesktop } from "../../../helpers/Breakpoints";
+
+import {Languages} from "../../../data/Languages";
+
+import {TouchHover} from "../../../helpers/TouchHover";
+import { LayoutContext, LayoutContextTypes } from "../../Layout/LayoutContext";
+import { translate } from "../../../helpers/translate";
+
+export class ServicesHeader extends React.Component<undefined, undefined> {
+    public static readonly contextTypes = LayoutContextTypes;
+
+    public context: LayoutContext;
+
+    public render(): JSX.Element {
+        return (
+            <header className="header header-services">
+                <Link to="/" className="logo">
+                    <i className="icon icon_logo"/>
+                    <span className="logo__text">Art &amp; Data Studio</span>
+                </Link>
+                <div className="header__right">
+                    <button type="button" className="header__lang-toggle" onClick={this.changeLanguage} {...TouchHover}>
+                        {this.languageLabel}
+                    </button>
+                    <button className="btn btn_close" />
+                </div>
+            </header>
+        );
+    }
+
+    protected changeLanguage = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
This is duplication of `Layout` component code.
It should be moved into separate component with tripple-language support